crates/web5/src/keys/key_manager/local_key_manager.rs

```diff
@@ -106,7 +106,7 @@ impl Default for LocalKeyManager {
 
 #[cfg(test)]
 mod tests {
-    use crate::key::Key;
+    use crate::keys::key::Key;
```

Comment:
later on we can think of a different names if people see this when they import from kotlin or something 😆

Reply:
[💯](https://emojipedia.org/hundred-points)